benchmark/runner/services.py

```
@@ -411,6 +425,47 @@ def cmd_start_adb_android_env(args: argparse.Namespace) -> int:
     return 0
 
 
+def cmd_start_desktop_env(args: argparse.Namespace) -> int:
+    if args.bridge_port < 0 or args.bridge_port > 65535:
+        print("Error: --bridge-port must be between 0 and 65535", file=sys.stderr); return 2
+    if args.ready_timeout_sec <= 0:
+        print("Error: --ready-timeout-sec must be positive", file=sys.stderr); return 2
+    service_id = _service_id("desktop", args.name)
+    service_dir = Path(args.runs_dir) / service_id; service_dir.mkdir(parents=True, exist_ok=True)
```

Ruff (0.16.2)

[error] Multiple statements on one line (semicolon)

(E702)